env0/resource_template_test.go

```diff
@@ -365,6 +365,50 @@ func TestUnitTemplateResource(t *testing.T) {
 		TerraformVersion:  "0.15.1",
 		IsAzureDevOps:     true,
 	}
+
+	helmTemplate := client.Template{
+		Id:               "helmTemplate",
+		Name:             "template0",
+		Description:      "description0",
+		Repository:       "env0/repo",
+		Path:             "path/zero/new",
+		Type:             "helm",
+		HelmChartName:    "chart1",
+		IsHelmRepository: true,
+		Retry: client.TemplateRetry{
+			OnDeploy: &client.TemplateRetryOn{
+				Times:      2,
+				ErrorRegex: "RetryMeForDeploy.*",
+			},
+			OnDestroy: &client.TemplateRetryOn{
+				Times:      1,
+				ErrorRegex: "RetryMeForDestroy.*",
+			},
+		},
+		TerraformVersion: "0.12.24",
```

Comment:
why `TerraformVersion`? you are testing helm templates

Reply:
This is a limitation in the acceptance tests. The use the same flow for all test-cases (Saves a lot of repetitive code).
I can fix it (by adding some 'ifs'), but I'm not sure it's worth the effort.

Reply:
we spoke on slack, we'll open a new issue for this matter